documentation/asciidoc/topics/proc_disabling-fips-mode-cr.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: PROCEDURE
+
+[id="proc_disabling-fips-mode-cr_{context}"]
+= Disabling FIPS mode in your {brandname} CR
+
+[role="_abstract"]
```

Comment:
no longer required for Jupiter,

Reply:
Yeah, I was not sure as I used the latest iteration of NewDoc and it was present in the auto-generated file.

Reply:
Newdoc is mostly used by the rhel writers and it's adapted to their needs/requirements